scripts/change-log-generator-utils.js

```diff
@@ -0,0 +1,274 @@
+const constants = require('./change-log-constants');
+const fs = require('fs');
+const shell = require('shelljs');
+const util = require('util');
+
+// Commit Map Keys
+const PR_NUM = 'PR_NUM';
+const LOG_HEADER = '# %s - %s\n';
+const COMMIT = 'COMMIT';
+const TYPE = 'TYPE';
+const MESSAGE = 'MESSAGE';
+const FILES_CHANGED = 'FILES_CHANGED';
+const RELEASE_DATE_ARG = '-t';
+const PACKAGES = 'PACKAGES';
+const TYPE_HEADER = '\n## %s\n';
+const SECTION_HEADER = '\n#### %s\n';
+const MESSAGE_FORMAT =
+  '\n- %s ([PR #%s](https://github.com/forcedotcom/salesforcedx-vscode/pull/%s))\n';
+const PR_ALREADY_EXISTS_ERROR =
+  'Filtered PR number %s. An entry already exists in the changelog.';
+
+const typesToIgnore = [
+  'chore',
+  'style',
+  'refactor',
+  'test',
+  'build',
+  'ci',
+  'revert'
+];
+
+module.exports = {
```

Comment:
should this go at the bottom?  In regular js land it's usually not good to use a thing before you declare it.

Reply:
Didn't know about that about exports. I can definitely move it to the bottom if that's a better JS practice.